examples/quickstart/main.rs

```diff
@@ -11,7 +11,7 @@ async fn age(
     ctx: Context<'_>,
     #[description = "Selected user"] user: Option<serenity::User>,
 ) -> Result<(), Error> {
-    let u = user.as_ref().unwrap_or(ctx.author());
```

Comment:
Is that a default clippy lint? Clippy doesn't complain when I run it locally (latest nightly)